packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButton.js

```diff
@@ -58,11 +58,25 @@ export const StyledDoneAndTalkButton = styled(Button)`
   `
 
 export function DoneAndTalkButton (props) {
-  function openTalkLinkAndClick(event) {
-    if (window && props.talkUrl) {
-      window.location = props.talkUrl
-    }
+  const openTalkLinkAndClick = function (event) {
+    const isCmdClick = event.metaKey
+
     props.onClick(event)
+      .then(() => {
+        if (window && props.talkURL) {
+          const url = `${window.location.origin}${props.talkURL}`
+          if (isCmdClick) {
+            event.preventDefault()
+            const newTab = window.open()
+            newTab.opener = null
+            newTab.location = url
+            newTab.target = '_blank'
+            newTab.focus()
+          } else {
+            window.location.assign(url)
+          }
+        }
```

Comment:
I'd feel happier if we just used a link, rather than writing our own code to imitate a link. I remember thinking that when I wrote the PFE code, though, so 🤷‍♂️.

Reply:
I dislike it too, but it's working... the hope is to remove it entirely post TESS.